src/Umbraco.Web.UI.Client/src/views/components/media/umb-media-node-info.html

```diff
@@ -12,9 +12,15 @@
 
                 <ul ng-if="nodeUrl" class="nav nav-stacked" style="margin-bottom: 0;">
                     <li>
-                        <a href="{{nodeUrl}}" target="_blank">
+                        <ng-conainer ng-if="node.extension != 'svg'">
+                            <a href="{{nodeUrl}}" target="_blank">
+                                <i class="icon icon-window-popin"></i>
+                                <span>{{nodeUrl}}</span>
+                            </a>
+                        </ng-conainer>
```

Comment:
What is `ng-conainer`? I guess it should be `ng-container`?
https://alligator.io/angular/ng-container-element/

Reply:
@bjarnef you are absolutely right! Nice spot 👍